galois_amd64.go

```diff
@@ -40,12 +40,12 @@ func galMulSSSE3Xor(low, high, in, out []byte) {
 }
 */
 
-func galMulSlice(c byte, in, out []byte, ssse3, avx2 bool) {
+func galMulSlice(c byte, in, out []byte, o options) {
```

Comment:
Thanks for this change. I guess whether sending `options` or `*options` depends on the cleverness of the compiler.

Could you check if there is any measurable difference?

Reply:
It is indeed better to pass a pointer, so will change that.